### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Googaa0082** [EvanJosephL/Harmony-RAT#1]
I followed every step and still nothing happens
The bot doesn't even come online
And please make a script to make exe file as the pyinstaller run into many problems
And in line 34 serverID there are no ('.') like in the other variables

**Comment by EvanJ4536** [comment]
> I followed every step and still nothing happens
> 
> The bot doesn't even come online
> 
> And please make a script to make exe file as the pyinstaller run into many problems
> 
> And in line 34 serverID there are no ('.') like in the other variables

If pyinstaller is having issues I think there may be other problems. 

The serverID variable is an integer, a big one. Don't put quotes.

**Comment by Googaa0082** [comment]
I tried it without the quotes at first 
But nothing happened
The bot didn't came online
Edit: can I get ur contact where you're usually online? Telegram perhaps?

**Comment by EvanJ4536** [comment]
Remove the "w" from ".pyw" and run harmony with python and see if the error is with the script, you can send the error here and I can try to help.

**Comment by Googaa0082** [comment]
![Screenshot_20241125_191108](https://github.com/user-attachments/assets/2a91b46c-521f-4baa-a3ef-793a1af1067f)

You mean remove 'w' from '.pyw' here or change the file extension.

**Comment by EvanJ4536** [comment]
Sorry, change the file extension.

**Comment by Googaa0082** [comment]
Brother it is not working
the bot is still offline
Plz contact me on telegram https://t.me/googaa12

**Comment by EvanJ4536** [comment]
I don't have telegram, if you want help it has to be here.

**Comment by Googaa0082** [comment]
I dont mind that but I did everything required and it still doesn't work. The bot dosent come online 
Put in all the variables, put the server id without quotes and changed the extension.
Im using python version 10 and I get no error.
Would making it into an (.exe) file work?

**Comment by EvanJ4536** [comment]
Does the script run when you run it with python?  If it does and your bot doesn't come online then either your bot is set up wrong or the variables are set wrong.

I wrote this script in Python 3, the different versions are 1, 2, and 3.

Converting to an exe wont work until the script runs with no errors.

I tested this script again the other day and it is working, did you make a text channel named "check-in" and assigned your webhook to post in that channel?  You need both a bot and  a webhook set up.

**Comment by Googaa0082** [comment]
I made a text channel named 'check-in' in my newly created server. I integeated webhook in that channel and paste its url in the rat. I made a bot and invite it to the server and paste its token in the rat. I put the server id in the rat without quotes. Saved the file and changed the file extension to py. Turned off antivirus. 
Still it doesn't work. 
When I run the file, command prompt opens for a bit then closes. No error is shown. 
And the bot is offline. 
I think its a library issue
